src/Catty/Defines/LanguageTranslationDefines.h

```diff
@@ -247,6 +247,7 @@
 #define kLocalizedLists NSLocalizedString(@"Lists", nil)
 #define kLocalizedPhiroBricks NSLocalizedString(@"Use Phiro bricks", nil)
 #define kLocalizedArduinoBricks NSLocalizedString(@"Arduino extension", nil)
+#define kLocalizedEmbroideryBricks NSLocalizedString(@"Use embroidery bricks", nil)
```

Comment:
Please rename to "Embroidery extension". And add the following subtitle/description to the settings page:

> Create patterns for stitching machines